### DISCUSSION TITLE
怎么搞配队啊

### DISCUSSION THREAD

**Discussion by Fufysui** [share121/inter-knot#4593] [绝区零]
不知道怎么配二队..
![1725767310944](https://github.com/user-attachments/assets/16bd5484-e0fa-497e-916a-291946b63658)

**Comment by fredericaina** [comment]
安比，丽娜，简

**Comment by skeleton114** [comment]
一看就是提前出的，一个赛斯没有

**Comment by peterquill163** [comment]
5
